Module-2/newspaper/index.html

```diff
@@ -0,0 +1,172 @@
+            <a href="https://www.knoe.com/contests/">Contests</a>
+        </div>
+    </nav>
+    <main>
+    <!-- This is for the ad space on the left side of the page-->
+    <div class="leftContainer" id="leftContainer">
+        
+    </div>
+
+    <!-- This is for the top middle container between left ad space and 7 day forcast-->
+    <div class="middleContainer" id="middleContainer"> 
+        <div class="top-left-image">
+            <img src="./assets/images/man.png" width="500" height="400">
+        </div>
+      
+    <!-- Middle column with 3 images vertical-->
+        <div class="column">
+            <div class="row">
+                <img src="./assets/images/weathermanImage.png" width="125" height="95">
+                <p>Knoe Thursday Evening Forcast Seasonal Friday, Heating Up Over the Weekend</p>
+            </div>
+            <div class="row">    
+                <img src="./assets/images/womanWithOpenMouth.png" width="125" height="95">
+                <p>House Republicans push off Biden impeachment move for now as hard-right clamors for action</p>
+            </div>
+            <div class="row">    
+                <img src="./assets/images/womanBehindFlag.png" width="125" height="95">
+                <p>A year after fall of Roe, 25 million women of childbearing age live in states with abortion bans or tigher restrictions</p>
+            </div>
+        </div>
+    </div>
+    <!-- Right column on page going down-->
+    
+    <div class="rightContainer">
+        <div class="weatherReport">
+            <p><strong>7 Day Forcast</strong></p>
+            <img src="./assets/images/weatherToday.png" width="250" height="150">
+            <p><strong>Share Videos & Photos</strong></p>
+            </div>
+        <div class="multiBoxes">
+            <p><strong>FEATURES</strong></p>
+            </div>
+    <div class="gridBox" id="gridBox">
+        <div class="boilAdvisories">
+            <img src="./assets/images/boil.png" width="100" height="50">
+            </div>
+        <div class="goodMorning">
+            <img src="./assets/images/goodMorning.png" width="100" height="50">
+            </div>
+        <div class="perfectHome">
+            <img src="./assets/images/perfectHome.png" width="100" height="50">
+            </div>
+        <div class="perfectHealth">
+            <img src="./assets/images/health.png" width="100" height="50">
+            </div>
+        <div class="ourTown">
+            <img src="./assets/images/ourTown.png" width="100" height="50">
+            </div>
+        <div class="mrFood">
+            <img src="./assets/images/mrFood.png" width="100" height="50">
+            </div>
+        <div class="serviceSalute">
+            <img src="./assets/images/serviceSalute.png" width="100" height="50">
+            </div>
+        <div class="talkingFood">
+            <img src="./assets/images/talkingFood.png" width="100" height="50">
+            </div>
```

Comment:
I'd recommend moving the `height` and width` properties of the images to a class that you then apply to these images - especially since many have the exact same height/width, that would reduce the amount of code in your HTML, and reduce the number of places you would need to update if you later decided to change the size of your images. 

This is also true for the other images on the page.